### PR TITLE
ISPN-12398 Update Backup manager resource names and directories

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.redhat.com/browse/ISPN-12398

### CODE REVIEW

File: cli/src/main/java/org/infinispan/cli/commands/Backup.java [hunk 1]
```diff
@@ -187,9 +187,9 @@ public boolean isHelp() {
             completer = CacheCompleter.class, name = CACHES)
       List<String> caches;
 
-      @OptionList(description = "Comma separated list of cache configurations to include, '*' indicates all available",
-            completer = CacheConfigurationCompleter.class, name = CACHE_CONFIGS)
-      List<String> cacheConfigs;
+      @OptionList(description = "Comma separated list of templates to include, '*' indicates all available",
```
Comment (oraNod) [inline cli/src/main/java/org/infinispan/cli/commands/Backup.java:190]:
```suggestion
      @OptionList(description = "Comma separated list of cache templates to include, '*' indicates all available",
```

File: cli/src/main/resources/help/backup.adoc [hunk 2]
```diff
@@ -32,17 +32,17 @@ Defines a name for the backup archive.
 *--caches*='cache1,cache2,...'::
 Lists caches to back up. Use '*' to back up all caches.
 
-*--cache-configs*='config1,config2,...'::
-Lists cache configurations to back up. Use '*' to back up all cache configurations.
+*--templates*='config1,config2,...'::
+Lists templates to back up. Use '*' to back up all templates.
```
Comment (oraNod) [inline cli/src/main/resources/help/backup.adoc:36]:
```suggestion
Lists cache templates to back up. Use '*' to back up all templates.
```

File: cli/src/main/resources/help/backup.adoc [hunk 3]
```diff
@@ -32,17 +32,17 @@ Defines a name for the backup archive.
 *--caches*='cache1,cache2,...'::
 Lists caches to back up. Use '*' to back up all caches.
 
-*--cache-configs*='config1,config2,...'::
-Lists cache configurations to back up. Use '*' to back up all cache configurations.
+*--templates*='config1,config2,...'::
```
Comment (oraNod) [inline cli/src/main/resources/help/backup.adoc:35]: Should this be 'template1,template2,...' ?

File: cli/src/main/resources/help/backup.adoc [hunk 4]
```diff
@@ -32,17 +32,17 @@ Defines a name for the backup archive.
...
 
 *--counters*='counter1,counter2,...'::
 Lists of counters to back up. Use '*' to back up all counters.
 
 *--proto-schemas*='schema1,schema2,...'::
 Lists Protobuf schemas to back up. Use '*' to back up all schemas.
 
-*--scripts*='script1,script2,...'::
-Lists scripts to back up. Use '*' to back up all scripts.
+*--tasks*='script1,script2,...'::
```
Comment (oraNod) [inline cli/src/main/resources/help/backup.adoc:44]:
```suggestion
*--tasks*='task1,task2,...'::
```

File: cli/src/main/resources/help/backup.adoc [hunk 5]
```diff
@@ -32,17 +32,17 @@ Defines a name for the backup archive.
...
+Lists tasks to back up. Use '*' to back up all tasks.
```
Comment (oraNod) [inline cli/src/main/resources/help/backup.adoc:45]:
```suggestion
Lists server tasks to back up. Use '*' to back up all tasks.
```

File: cli/src/main/resources/help/backup.adoc [hunk 6]
```diff
@@ -62,17 +62,17 @@ Defines a name for the restore request.
 *--caches*='cache1,cache2,...'::
 Lists caches to restore. Use '*' to restore all caches from the backup archive.
 
-*--cache-configs*='config1,config2,...'::
-Lists cache configurations to restore. Use '*' to restore all cache configurations from the backup archive.
+*--templates*='config1,config2,...'::
+Lists templates to restore. Use '*' to restore all templates from the backup archive.
```
Comment (oraNod) [inline cli/src/main/resources/help/backup.adoc:66]:
```suggestion
Lists cache templates to restore. Use '*' to restore all templates from the backup archive.
```

File: cli/src/main/resources/help/backup.adoc [hunk 7]
```diff
@@ -62,17 +62,17 @@ Defines a name for the restore request.
...
 *--counters*='counter1,counter2,...'::
 Lists counters to restore. Use '*' to restore all counters from the backup archive.
 
 *--proto-schemas*='schema1,schema2,...'::
 Lists Protobuf schemas to restore. Use '*' to restore all schemas from the backup archive.
 
-*--scripts*='script1,script2,...'::
-Lists scripts to restore. Use '*' to restore all scripts from the backup archive.
+*--tasks*='script1,script2,...'::
+Lists tasks to restore. Use '*' to restore all tasks from the backup archive.
```
Comment (oraNod) [inline cli/src/main/resources/help/backup.adoc:75]:
```suggestion
Lists server tasks to restore. Use '*' to restore all tasks from the backup archive.
```

File: cli/src/main/resources/help/backup.adoc [hunk 8]
```diff
@@ -62,17 +62,17 @@ Defines a name for the restore request.
...
 
 *--counters*='counter1,counter2,...'::
 Lists counters to restore. Use '*' to restore all counters from the backup archive.
 
 *--proto-schemas*='schema1,schema2,...'::
 Lists Protobuf schemas to restore. Use '*' to restore all schemas from the backup archive.
 
-*--scripts*='script1,script2,...'::
-Lists scripts to restore. Use '*' to restore all scripts from the backup archive.
+*--tasks*='script1,script2,...'::
```
Comment (oraNod) [inline cli/src/main/resources/help/backup.adoc:74]:
```suggestion
*--tasks*='task1,task2,...'::
```

File: documentation/src/main/asciidoc/topics/backup_resources.adoc [hunk 9]
```diff
@@ -8,7 +8,7 @@
 
 |`cache-configs`
 |OPTIONAL
-|Specifies either an array of cache configurations to back up or `*` for all cache configurations.
+|Specifies either an array of templates to back up or `*` for all templates.
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/backup_resources.adoc:11]:
```suggestion
|Specifies either an array of cache templates to back up or `*` for all templates.
```

File: documentation/src/main/asciidoc/topics/backup_resources.adoc [hunk 10]
```diff
@@ -18,8 +18,8 @@
 |OPTIONAL
 |Defines either an array of Protobuf schema names to back up or `*` for all schemas.
 
-|`scripts`
+|`tasks`
 |OPTIONAL
-|Specifies either an array of script names to back up or `*` for all scripts.
+|Specifies either an array of script names to back up or `*` for all tasks.
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/backup_resources.adoc:23]:
```suggestion
|Specifies either an array of server tasks to back up or `*` for all tasks.
```

File: documentation/src/main/asciidoc/topics/proc_cli_backup_creating.adoc [hunk 11]
```diff
@@ -35,11 +35,11 @@ create backup archives.
 [//containers/default]> backup create -d /some/server/dir
 ----
 +
-* Back up only caches and cache configurations.
+* Back up only caches and templates.
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/proc_cli_backup_creating.adoc:38]:
```suggestion
* Back up only caches and cache templates.
```

File: documentation/src/main/asciidoc/topics/ref_rest_cache_managers.adoc [hunk 12]
```diff
@@ -210,7 +210,7 @@ include::json_examples/rest_cache_stats_response.json[]
 [id='rest_v2_backup_manager_create']
 = Backing Up {brandname} Cache Managers
 Create backup archives, `application/zip`, that contain resources (caches, cache
-configurations, counters, Protobuf schemas, scripts, and so on) currently
+templates, counters, Protobuf schemas, tasks, and so on) currently
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/ref_rest_cache_managers.adoc:213]:
```suggestion
cache templates, counters, Protobuf schemas, server tasks, and so on) currently
```

File: documentation/src/main/asciidoc/topics/ref_rest_clusters.adoc [hunk 13]
```diff
@@ -25,9 +25,8 @@ POST /rest/v2/cluster?action=stop&server={server1_host}&server={server2_host}
 
 [id='rest_v2_backup_cluster']
 = Backing Up {brandname} Clusters
-Create backup archives, `application/zip`, that contain resources (caches, cache
-configurations, counters, Protobuf schemas, scripts, and so on) currently
-stored in the cache container for the cluster.
+Create backup archives, `application/zip`, that contain resources (caches, templates, counters, Protobuf schemas, tasks,
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/ref_rest_clusters.adoc:28]:
```suggestion
Create backup archives, `application/zip`, that contain resources (caches, cache templates, counters, Protobuf schemas, server tasks,
```

**Comment by ryanemerson** [comment]
All addressed, thanks @oraNod

**Review by oraNod — APPROVED** [review]
LGTM. Thanks, Ryan!

**Review by karesti — CHANGES_REQUESTED** [review]
may this change have impacted some tests ?

**Comment by tristantarrant** [comment]
@ryanemerson you need to fix the path in BackupManagerIT

**Comment by ryanemerson** [comment]
Should be good now

**Comment by wburns** [comment]
Integrated into master, thanks @ryanemerson !

**Pull request merged**